src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -125,10 +125,28 @@
     "drop": PREFIX + 239,
     "decision": PREFIX + 240,
     "keep": PREFIX + 241,
+    # Spans
+    "span.op": PREFIX + 242,
+    "span.module": PREFIX + 243,
+    "span.action": PREFIX + 244,
+    "span.system": PREFIX + 245,
+    "span.status": PREFIX + 246,
```

Comment:
I am not too familiar with what `SHARED_TAG_STRINGS` are. @ayirr7 could you provide some context? Are these supposed to be shared between the release health and generic metrics pipeline? If yes, the span metrics are going to be used on the generic metrics pipeline only. How do you create tag strings belonging to a specific pipeline?